book/src/confidence.md

```diff
@@ -0,0 +1,67 @@
+# Confidence
+
+The confidence metric aims to give clients a measure of the network confirmation levels on a particular block.
+
+# Calculation RPC
+
+Clients can request confidence metrics from a validator for a signature `s` through `get_confidence(s: Signature) -> Confidence` over RPC. The `Confidence` struct is of type `HashMap<u32, f32>`. This returned hashmap represents the confidence metric for the particular block `N` that contains the signature `s` as of the last block `M` that the validator voted on. 
+
+An entry `(i, p)` in the `Confidence` hashmap implies that the validator observed `p` percentage of the network reaching `i` confirmations on block `N` as observed in some block `M`. There will be `MAX_LOCKOUT` entries in this map, representing all the possible number of confirmations from 1 and to `MAX_LOCKOUT`.
```

Comment:
`i` confirmations is more like 'i` blocks back?  why not return a vector of 'p' from `M` to `N`?

Reply:
Yeah that would work!